react/modules/enhancedEcommerceEvents.ts

```diff
@@ -71,6 +75,7 @@ export async function sendEnhancedEcommerceEvents(e: PixelMessage) {
                 dimension2: skuReferenceId ?? '',
                 dimension3: selectedSku.name,
                 price,
+                isAvailable,
```

Comment:
I don't think it should be called isAvailable.
```suggestion
                dimension4: isAvailable,
```

Reply:
@icazevedo I changed the value of the property to isAvailable. Given that it was a boolean I also made the possible values available or unavailable, this way it is more descriptive for the customers what do you think?
https://tagmanager--arcaplanet.myvtex.com/monge-medium-adult-ricco-di-pollo/p -> available product
https://tagmanager--arcaplanet.myvtex.com/seresto-collare-antiparassitario-cani-oltre-8-kg/p -> unavailable product